test/lib/graph/graph.integration.js

```diff
@@ -253,7 +253,8 @@ test.serial("graphFromPackageDependencies with inactive workspace file at custom
 		rootConfigPath: "/rootConfigPath",
 		versionOverride: "versionOverride",
 		workspaceName: "default",
-		workspaceConfigPath: path.join(libraryHPath, "custom-ui5-workspace.yaml")
+		workspaceConfigPath: path.join(libraryHPath, "custom-ui5-workspace.yaml"),
+		cacheMode: "force"
```

Comment:
Should be an upper-case `F`, right? Or ideally using the enum object.

Reply:
Done